tests/app/v2/notifications/test_get_notifications.py

```diff
@@ -295,6 +295,34 @@ def test_get_all_notifications_returns_200(client, sample_template):
     assert not json_response['notifications'][0]['scheduled_for']
 
 
+def test_get_all_notifications_with_include_jobs_arg_returns_200(
```

Comment:
Do we have a test that checks that notifications created by a job are not included when the param does not exist?

Reply:
ok, I see what you mean, can add it